packages/blade/src/components/Collapsible/CollapsibleButton.tsx

```diff
@@ -0,0 +1,53 @@
+import type { ReactElement } from 'react';
+import { useCallback } from 'react';
+import { useCollapsibleContext } from './CollapsibleContext';
+import type { ButtonProps } from '~components/Button';
+import { Button } from '~components/Button';
+
+type CollapsibleButtonProps = Pick<
+  ButtonProps,
+  | 'variant'
+  | 'size'
+  | 'iconPosition'
+  | 'isDisabled'
+  | 'testID'
+  | 'accessibilityLabel'
+  | 'icon'
+  | 'children'
+>;
+
+// TODO: ask anurag about this type problem
+
+const CollapsibleButton = ({
```

Comment:
We might need to expose ref?

Reply:
Handled in https://github.com/razorpay/blade/pull/1296